automated install/basic-install.sh

```diff
@@ -1677,7 +1689,8 @@ installPiholeWeb() {
     # Make the .d directory if it doesn't exist
     mkdir -p /etc/sudoers.d/
     # and copy in the pihole sudoers file
-    cp ${PI_HOLE_LOCAL_REPO}/advanced/Templates/pihole.sudo /etc/sudoers.d/pihole
+    # set the strict permissions on the file
+    install -m 0440 ${PI_HOLE_LOCAL_REPO}/advanced/Templates/pihole.sudo /etc/sudoers.d/pihole
     # Add lighttpd user (OS dependent) to sudoers file
     echo "${LIGHTTPD_USER} ALL=NOPASSWD: /usr/local/bin/pihole" >> /etc/sudoers.d/pihole
```

Comment:
If the read-only permissions are given to the file, and then we try to write to the file, the write will fail. Strict permissions need to be set after we are done modifying the file.

Reply:
Fixed.